app/code/Magento/Deploy/Console/DeployStaticOptions.php

```diff
@@ -137,6 +137,11 @@ class DeployStaticOptions
      */
     const CONTENT_VERSION = 'content-version';
 
+    /**
+     * Key for languages parameter
+     */
+    const DELETE_PREVIOUS_FILES = 'delete-previous-files';
```

Comment:
Please, make const private

Reply:
It is used  in DeployStaticContentCommand, if I make it private it will not be accesible